src/components/Pagination/Pagination.scss

```diff
@@ -12,13 +14,13 @@ $prefixClass: rc-pagination;
 }
 
 .rc-pagination {
-  font-size: 12px;
+  font-size: $font-size;
   font-family: 'Arial';
   user-select: none;
   padding: 0;
-  display: flex;
+  display: flex; 
 
-  > li {
+  > li {   
```

Comment:
@Farhan-emgage try not to add extra spaces. Sometimes the linter complains about this.